tests/test_lambdas/test_client.py

```diff
@@ -3,21 +3,36 @@
 import json
 import logging
 from io import BytesIO
-from unittest.mock import MagicMock
+from unittest.mock import ANY, MagicMock
 
 import pytest
 from pytest import LogCaptureFixture
 from pytest_mock import MockerFixture
 
 from lbz.aws_boto3 import Boto3Client
-from lbz.lambdas import LambdaClient, LambdaError, LambdaResult
+from lbz.lambdas import LambdaClient, LambdaError, LambdaResult, LambdaSource
 
 
 @pytest.fixture(name="lambda_client")
 def lambda_client_fixture(mocker: MockerFixture) -> MagicMock:
     return mocker.patch.object(Boto3Client, "lambda_")
 
 
+def rest_response_factory() -> dict:
+    return {
+        "Payload": BytesIO(
+            json.dumps(
+                {
+                    "body": {},
+                    "statusCode": 200,
+                    "headers": {},
+                    "isBase64Encoded": False,
```

Comment:
As I mentioned, it would be good to make this data less default:
```suggestion
                    "body": {"message": "Hello World!"},
                    "statusCode": 200,
                    "headers": {"Content-Type": "application/json"},
                    "isBase64Encoded": False,
```